src/ventus/wm/windowmanager.js

```diff
@@ -199,12 +217,12 @@ function($, Window, View, DefaultMode, ExposeMode, FullscreenMode) {
 
 	WindowManager.prototype.createWindow.fromQuery = function(selector, options) {
 		options.content = View(selector);
-		return this.createWindow(options);
+		return this.createWindow(this, options);
```

Comment:
Roger. I missed that. Line 220, 225 should not be there it was oversight on my part. My idea was for `createWindow` to handle passing the reference to the newly created window.